algorithms/CSharp/test/Strings/character-limit.cs

```diff
@@ -0,0 +1,20 @@
+using NUnit.Framework;
+
+namespace Algorithms.CSharp.Test.Strings
+{
+    [TestFixture]
+    internal class Character_Limit
+    {
+        [TestCase("Hello", "Hel..")]
+        [TestCase("World", "Wor..")]
+        [TestCase("How Ya", "How...")]
+        [TestCase("Doin", "Doi.")]
+
+
+        public void PassString_ShouldGetExpectedResult(string text, string expected)
+        {
+            string result = Algorithms.Strings.TextHelper.Character_Limit(text, 3);
+            Assert.IsTrue(result == expected, "error");
```

Comment:
Please use `Assert.AreEqual`
```suggestion
            Assert.AreEqual(expected, result);
```